tests/integration_tests/modules/test_lxd.py

```diff
@@ -22,7 +22,7 @@
     ipv4_netmask: 24
     ipv4_dhcp_first: 10.100.100.100
     ipv4_dhcp_last: 10.100.100.200
-    ipv4_nat: true
+    ipv4_nat: "true"
```

Comment:
Why the change? `cloud-init schema --system` complains about this not being a boolean (causing the log check to fail 
 in test_lxd.py::TestLxdBridge::test_bridge)